### PR TITLE
[Hub Generated] Review request for Microsoft.AzureActiveDirectory to add version preview/2019-01-01-preview

### PR DESCRIPTION
**Pull request by nickgmicrosoft**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/nickgmicrosoft/azure-rest-api-specs/dev-cpim-Microsoft.AzureActiveDirectory-2019-01-01-preview-australia...master/).

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by ChenTanyi** [comment]
@msyyc @jsntcy Can you take a look at the python generation error?

File: specification/cpim/resource-manager/readme.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ These are the global settings for the azureadb2c APIs.
 title: CPIMConfigurationClient
 description: CPIM Configuration Client 
 openapi-type: arm
-tag: package-2020-05-01-preview
+tag: package-2019-01-01-preview
```
Comment (ChenTanyi) [inline specification/cpim/resource-manager/readme.md:31]: why to decrease the default version?

**Comment by nickgmicrosoft** [comment]
> @msyyc @jsntcy Can you take a look at the python generation error?

From the [original PR](https://github.com/Azure/azure-rest-api-specs/pull/11292):
image

File: specification/cpim/resource-manager/readme.md (hunk 1 above)
Comment (nickgmicrosoft) [reply specification/cpim/resource-manager/readme.md:31]: The cli team is working on sdk generation for the 2019-01-01-preview version, not the 2020-05-01-preview. We're prioritization 2019-01-01-preview at this time.

**Comment by ChenTanyi** [comment]
I think the python is already changed to track2. The error comes from track2 generation.

**Comment by nickgmicrosoft** [comment]
> I think the python is already changed to track2. The error comes from track2 generation.

There's a separate line item for track 2, see:
image.

I think we had a similar issue with the original PR of the validator still showing an error. Maybe the tooling needs to be updated?

**Review by ChenTanyi — APPROVED** [review]

**Pull request merged**